### PR TITLE
fix: Updated homepage to give more info about website

### PR DESCRIPTION
**Pull request by harini-1597**
## Related Issue
Fixes https://github.com/Sanchitbajaj02/palettegram/issues/285

## Description
I feel that the website home page doesn't really tell much about what the website is for and only asks you to sign in leading to lesser user retention;
I would like the home-page scroll to offer some more information as user scrolls down:
-> What is Palettegram?
-> Who is Palettegram for? 
-> Why Palettegram?
(and some other useful info...)

## Screenshots
Here's a video preview of the proposed changes as discussed with @Sanchitbajaj02:

https://github.com/Sanchitbajaj02/palettegram/assets/128628820/a443d1df-8482-4603-941f-f406b760da89

## Summary by CodeRabbit

- **New Features**
  - Enhanced user experience with motion effects on headings and paragraphs on the home page.
  - Improved text content to better communicate the platform's purpose and target audience.
  
- **Style**
  - Updated image sources on the home page for a refreshed look.
  - Imported Google Fonts and added new CSS rules for enhanced styling.

### CODE REVIEW

**Comment by harini-1597** [comment]
#285

File: src/styles/globals.css [hunk 1]
```diff
@@ -1,6 +1,7 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
```
Comment (Sanchitbajaj02) [inline src/styles/globals.css:4]: Poppins is already configured in the src/app/layout.tsx. Please use that as this method is redundent

File: src/styles/globals.css [hunk 2]
```diff
@@ -41,6 +42,9 @@ body {
   .custom-shadow {
     filter: drop-shadow(0 5px 150px rgba(182, 0, 85, 0.75));
   }
+  .sub-section {
```
Comment (Sanchitbajaj02) [inline src/styles/globals.css:45]: Why is this added?

File: src/components/pages/home/index.tsx [hunk 3]
```diff
@@ -99,7 +99,93 @@ function HomePage({ accountId }: { accountId: string | undefined }) {
...
+              viewport={{ once: true }}
+              transition={{
+                duration: 1,
+                type: "spring",
+                stiffness: 80,
+                delay: 0,
+              }}
+              className="text-3xl md:text-6xl text-center font-extrabold tracking-wide text-black dark:text-white md:text-right"
+            >
+              What is <br /> Palettegram?
```
Comment (Sanchitbajaj02) [inline src/components/pages/home/index.tsx:124]: No need to add br tag. Please remove it

File: src/components/pages/home/index.tsx [hunk 4]
```diff
@@ -121,7 +232,7 @@ function HomePage({ accountId }: { accountId: string | undefined }) {
               }}
               className="text-3xl md:text-6xl text-center font-extrabold tracking-wide text-black dark:text-white md:text-right"
             >
-              Who is <br /> Palettegram for?
+              Create, Showcase <br /> Collaborate
```
Comment (Sanchitbajaj02) [inline src/components/pages/home/index.tsx:235]: No need to add br tag. Please remove it

**Review by Sanchitbajaj02 — CHANGES_REQUESTED** [review]

**Comment by harini-1597** [comment]
@Sanchitbajaj02 I've made the changes requested

**Comment by yashd-dev** [comment]
Hey there, the illustrations used do not have a transparent background which makes it look out of place in dark mode, kindly use transparent illustrations.

image

**Comment by harini-1597** [comment]
Are these okay?
Or would you like them all gray? Personally, I feel this adds more color to the website, what do you think?

Screen Shot 2024-05-19 at 11 09 55 AM


Screen Shot 2024-05-19 at 11 10 12 AM

**Review by Sanchitbajaj02 — APPROVED** [review]
LGTM

**Pull request merged**